src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -183,7 +215,7 @@ private RequestContext CreateRequestContext(Guid telemetryCorrelationId)
         /// specify extra query parameters
         ///
         /// </remarks>
-        public AcquireTokenSilentParameterBuilder AcquireTokenSilent(IEnumerable<string> scopes, IAccount account)
+        public AcquireTokenSilentParameterBuilder AcquireTokenSilentWithAccount(IEnumerable<string> scopes, IAccount account)
```

Comment:
>AcquireTokenSilentWithAccount [](start = 50, length = 29)

I thought we agreed not to change this to ensure we don't break people? If we go on with this proper migration aid should be provided ensuring the compiler will say exactly what needs to be done.

Reply:
I second Henrik. I thought we don't want to change the name and signature of existing methods. I would really not want us to break anybody here. just bring a bit of comfort.